src/presets/netlify/utils.ts

```diff
@@ -92,6 +92,13 @@ export async function writeHeaders(nitro: Nitro) {
   await fsp.writeFile(headersPath, contents);
 }
 
+export function getStaticPaths(nitro: Nitro): string[] {
+  const publicAssets = nitro.options.publicAssets.filter(
+    (dir) => dir.fallthrough !== true && dir.baseURL && dir.baseURL !== "/"
```

Comment:
@pi0 is this `dir.baseUrl !== "/"` case right? I was seeing this be included when running the tests and I saw similar exclusions in some other presets. I'm not sure I understand what this is doing though.

Reply:
It is correct. The check is to make sure we don't include root `/` patterns (that imply fallthrough behavior) to exclude. Nitro usually infers `fallthrough` properly but it might not be set correctly (ie, after creating nitro instance via wrapper frameworks)